### ISSUE TITLE
Bug Hunting!!

### ISSUE THREAD

**Issue by liangyiliang** [codex-studio/nav#9]
Hi all, 
If you guys see me writing anything in GitHub, it is most likely bug reports. 
1. When the search box has nothing inside and I click the search button **twice in a second**, the notification of Search cannot be empty! shows up twice, just like this. I think it is the best to show it only once at a time. 
<img width="238" alt="snip_20160716192508" src="https://cloud.githubusercontent.com/assets/16521252/16894622/56b3751e-4b8e-11e6-84ba-b2b47f4998a2.png">
2. Why is there a little dot besides the _TUMBLR_ link?
3. Just below the CSDS Logo, there is a button of _Edit Name_ or _Add Name_ with a picture (or shall I call it a symbol). The texts are NOT aligned with the symbol. 
4. The date disappears from the top-right corner of the page when the window’s width is shortened. 
5. My computer is configured in English, therefore I prefer to use to use the English version of CodEX Nav. However, when the browser is just opened, I can see some Chinese on the title bar which means “CodEX Navigation”. After a few seconds it changes to English automatically. 
6. When the button _Changelog_ is pressed, it navigates to a completely different page without any animation. I recommend to make it a pop-up style frame that shows the changelog. When anywhere else on the page is clicked, the pop-up disappears. 

These are what I can find on CodEX Nav right now. If I found more I will write another issue.
The bug reports I made today, Excited! [-]..[-]

Leo Liang 
onewindows@codex.studio 
7/16/2016

**Comment by mj66** [comment]
Hi Leo: 
Thanks for hunting all these bugs. However there are some bugs that are technically not "bugs". Here's the detail for each one of those:
1. "Search cannot be empty" notification is shown by the notification slider, which is a list of text and notifications just like the ones you can see on Windows 10 and Android and iOS. They are accumulated, which means several of them can appear at a time, depending on the frequency of callouts. There is no way that we can fix this, sorry about that.
2. Tumblr. I know that tumblr's legal name is Tumblr, Inc., but in the media use logo documentation, there is a description as followings:
> **Don’t add or remove the period.**
The little "t" logo should never have a period. The full "tumblr." logo should never be without.
![capture](https://cloud.githubusercontent.com/assets/13333578/16895201/6912817a-4b3c-11e6-80d0-6d93728425f4.PNG)

Therefore, we decided to leave the period to respect the logo and the name, as we can not use logos in our buttons.
3. Yes, this could be a problem. I'll try my best to keep them aligned.
![capture](https://cloud.githubusercontent.com/assets/13333578/16895210/dbd478f8-4b3c-11e6-9123-d4872279a748.PNG)
4. In Build 200009 [Commit 9565a08](https://github.com/codex-studio/nav/commit/9565a089b736b8116432fe826cf91d2ebb559732), we added a box on the bottom of the page to replace the disapperance of the date box. We are trying to make a bar showing below the main navbar after the screen has been resized, and it won't take long.
5. Yes, this is a known problem and we will try our best to configure the sequence of loading the page. 
6. Thanks for suggesting that. This is a great idea. We are going to do it. Last time when me and @casprussian are doing it we found out a little problem with the popup box so we decided to put the idea aside. Soon i found out it was because of a Jquery version problem and after I upgraded the jquery version in Build 200019 [Commit ca3c4dd](https://github.com/codex-studio/nav/commit/ca3c4dd0c5289d5e23a626807eea469f661660c7) the popup box can be displayed correctly. We will be working on it right now. 

Hope all these helps. Peace. :smile: 
Mingjie Jiang - Core Developer
jimmy@codex.studio
7/16/2016

**Comment by casprussian** [comment]
@mj66 
For bug #6, you said that we can't have that because of the way that Materialize.CSS works.

**Comment by mj66** [comment]
@casprussian Good thing is, I updated jquery and found out that it works only in certain versions of jquery. I actually posted an issue and ASKED the materializecss team. Try it out [here](http://nav.codex.studio)

**Comment by mj66** [comment]
@liangyiliang All mentioned fixed in [Build 200020](https://github.com/codex-studio/nav/commit/bd305f9e62f8710ea9560f12bfa8cd59afe1cfcd)
Thanks.
